www/src/views/starter-library/filtered-starters.js

```diff
@@ -66,7 +66,10 @@ export default class FilteredStarterLibrary extends Component {
       )
     )
 
-    let starters = data.allStartersYaml.edges
+    // filter out starters missing github information
+    let starters = data.allStartersYaml.edges.filter(({ node: starter }) => {
```

Comment:
Could we add the filtering at the [GraphQL layer](https://github.com/gatsbyjs/gatsby/blob/d77d4eeeabdd8f695c3aa8628273ff8fcb486297/www/src/pages/starters.js#L19)? Then we don't have to worry about ever getting "bad" data?

Reply:
So this is part of the problem. There are like 1-2 starters that have data at the GraphQL layer, that somehow gets stripped in the view. I'm going to account for that in a new issue, but this is to get the view to not break in the meantime.

Reply:
Yeah now I'm not even seeing it on prod or local. This is a decent stopgap i think.

Reply:
With this approach or the GraphQL filtering?

Reply:
Either. This hasn't been merged, so prod is just prod, as it was when this was originally reported -- which makes me really not understand the inconsistent recurrence of this bug. I also don't see it reflected on my local branch. Also the starters missing data (sometimes) vary.

Reply:
I'm fine with leaving this stopgap here, and i'm going to make a hacktoberfest issue to investigate the disparity